### ISSUE TITLE
resnet50.onnx --> tflite Convert Error

### ISSUE THREAD

**Issue by JunHyunjae** [tensorflow/tensorflow#51522]
### 1. System information
pip3 install tensorflow-gpu==2.4.0
pip3 install tensorflow-addons==0.13.0
onnx == 1.7.0

onnx model link (https://github.com/onnx/models/blob/master/vision/classification/resnet/model/resnet50-v1-7.onnx)

Our model uses backbone as Resnet-50.
The error occurred in our model, so even if we downloaded the official onnx model and converted it, it shows the same result.
In 2.2.0 and 2.3.0, the conversion is good.
And I have to use version 2.4.0.

If converter.allow_custom_ops is True, it is converted, but eventually problems occur at runtime.

### 2. Code

Provide code to help us reproduce your issues using one of the following options:
# onnx --> pb
onnx_model = onnx.load("./resnet50-v1-7.onnx")
tf_model_path = "./saved_model"
tf_rep = prepare(onnx_model)
tf_rep.export_graph(tf_model_path)

# pb --> tflite
converter = tf.lite.TFLiteConverter.from_saved_model(tf_model_path)
tflite_model = converter.convert()
tflite_model_path = "./onnx_backbone/tf_2.4.0/tflite/backbone_v9_default.tflite"
with open(tflite_model_path, 'wb') as f:
    f.write(tflite_model)

```
(You can paste links or attach files by dragging & dropping them below)
- Provide links to your updated versions of the above two colab notebooks.
- Provide links to your TensorFlow model and (optionally) TensorFlow Lite Model.
```

#### Option B: Paste your code here or provide a link to a custom end-to-end colab

```
(You can paste links or attach files by dragging & dropping them below)
- Include code to invoke the TFLite Converter Python API and the errors.
- Provide links to your TensorFlow model and (optionally) TensorFlow Lite Model.
```

### 3. Failure after conversion
In 2.4.0, the following error occurs:

loc(callsite(callsite("MaxPool2d@__inference___call___1278" at "PartitionedCall@__inference_signature_wrapper_1497") at "PartitionedCall")): error: 'tf.MaxPool' op is neither a custom op nor a flex op
error: failed while converting: 'main': Ops that can be supported by the flex runtime (enabled via setting the -emit-select-tf-ops flag):
	tf.MaxPool {data_format = "NCHW", device = "", explicit_paddings = [], ksize = [1, 1, 3, 3], padding = "VALID", strides = [1, 1, 2, 2]}
Traceback (most recent call last):
  File "/workspace/venv/tensorflow2_4/lib/python3.6/site-packages/tensorflow/lite/python/convert.py", line 213, in toco_convert_protos
    enable_mlir_converter)
  File "/workspace/venv/tensorflow2_4/lib/python3.6/site-packages/tensorflow/lite/python/wrap_toco.py", line 38, in wrapped_toco_convert
    enable_mlir_converter)
Exception: <unknown>:0: error: loc(callsite(callsite("MaxPool2d@__inference___call___1278" at "PartitionedCall@__inference_signature_wrapper_1497") at "PartitionedCall")): 'tf.MaxPool' op is neither a custom op nor a flex op
<unknown>:0: note: loc("PartitionedCall"): called from
<unknown>:0: error: failed while converting: 'main': Ops that can be supported by the flex runtime (enabled via setting the -emit-select-tf-ops flag):
	tf.MaxPool {data_format = "NCHW", device = "", explicit_paddings = [], ksize = [1, 1, 3, 3], padding = "VALID", strides = [1, 1, 2, 2]}

**Comment by abattery** [comment]
Please consider using the Select TF option.

**Comment by JunHyunjae** [comment]
TF 2.5.0 is now available on the chip. So we solved it by converting from 2.5.0.
